src/main.rs

```diff
@@ -516,4 +523,51 @@ mod tests {
     drop(test_input_file);
     temp_dir.close().expect("Delete test directory");
   }
+
+  #[test]
+  fn creates_html_in_directory() {
+    let temp_dir = tempfile::tempdir().unwrap();
+    let test_input_path1 = temp_dir.path().join("html_template_test1.txt");
+    let test_input_path2 = temp_dir.path().join("html_template_test2.txt");
+    // let test_input_path_string1 = test_input_path1.as_os_str().to_str().unwrap().to_string();
+    // let test_input_path_string2 = test_input_path2.as_os_str().to_str().unwrap().to_string();
+    let mut test_input_file1 = File::create(&test_input_path1).unwrap();
+    let mut test_input_file2 = File::create(&test_input_path2).unwrap();
+
+    writeln!(test_input_file1, "test1").expect("Create test input file");
+    writeln!(test_input_file2, "test2").expect("Create test input file");
+
+    let input_dir = fs::read_dir(&temp_dir).expect("Read input directory");
+    let out_dir = "./dist".to_owned();
+    convert_files_in_directory(input_dir, &out_dir, "en");
```

Comment:
When convert_files_in_directory is called here, the output directory has not been created
You should create the directory first before the function call.
Also, please delete it at the end of the test too

Reply:
On second thought, it is probably best to rename the output directory too since it could interfere with the recreates_default_output_directory test

Reply:
Thanks for the feedback. It's strange that it passes in my local sometimes. I think multithread is running here. Let me figure it out how I should avoid this.

Reply:
Yes, they are run in parallel (multithreading) like you suspect. There is another test that creates the same output directory which is why they would pass sometimes

Reply:
ah, that makes sense. Not sure about the reason, but even single thread test with `-- --test-threads=1` does not work. Let me just remove `create_files_in_directory` test until I figure out something. Thanks for your support.

Reply:
@Eakam1007 Would you please merge current branch? I don't think I can write another test any time soon. Please let me know what you think.